### ISSUE TITLE
No item drop on pulling out Phone

### ISSUE THREAD

**Issue by Agitatio** [CassCoffey/LethalPhones#1]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

I can understand it happening when hands are full, but in all other cases it just becomes an annoyance. Especially when coupled with something like reserved slots mods, items from which also get force dropped. Hopefully this can be remedied or made an option.

While I'm here I would also like to suggest two other things. Sound for switching to vibrate mode, and sounds for taking phone out and putting it back in. Like flip open/close sounds?

Edit: after actually using it for a bit and thinking it through, there are the changes that are in my option will improve and balance the phone a bit more:
- Pulling it out should only be possible when hands aren't full. Item that was previously in hand is simply hidden similar to switching to empty slot and not dropped. Possibly switched back to after putting phone away.
- Only usable while in hand. No talking while its in your pocket. Putting it away should hang up a call. Makes walkies even more useful in comparison.

**Comment by probskay** [comment]
have you ever tucked a phone between your shoulder and face while making a phone call? i think 'pocket calls' are fine

**Comment by Agitatio** [comment]
Okay, let's assume that. It still makes it a bit too useful compared to a walkie.

**Comment by VanilaBOI** [comment]
I totally agree with @probskay that you can still continue a call while holding a two handed item.
But way use a walkie at that point. As long as it is a config option I think it would only improve the mod.

**Comment by CassCoffey** [comment]
There's now a config option to auto-hangup the phone when putting it away, as of [this commit](https://github.com/CassCoffey/LethalPhones/commit/ba021f5b6f03b7f8eab2fdcf730642262bf73e67).

Item dropping was fixed a while back as well.
